corehq/apps/es/migrations/0012_add_new_index_for_bha.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.2.11 on 2024-05-10 14:28
+
+from django.conf import settings
+from django.db import migrations
+
+from corehq.apps.es.mappings.case_search_mapping import CASE_SEARCH_MAPPING
+from corehq.apps.es.migration_operations import (
+    CreateIndex,
+    DeleteOnlyIfIndexExists,
+)
+from corehq.apps.es.utils import index_runtime_name
+
+
+def _create_bha_index_on_saas_envs(apps, schema_editor):
+    if settings.IS_SAAS_ENVIRONMENT:
```

Comment:
I could've swarn India was also a SaaS environment, but it looks like this is actually just prod + staging

Reply:
Yeah. I also used to think that way until I stumbled upon it 😅